openapi/task_execution_service.openapi.yaml

```diff
@@ -654,6 +669,12 @@ components:
           type: string
           description: |-
             Optional user-provided description of task for documentation purposes.
+        callback_url:
+          type: string
+          format: uri
+          description: |-
+            Optional user-provided callback URL for task status updates.
```

Comment:
Note that this should probably _not_ be shared together with other info when users call, e.g., `GET /tasks/{task_id}`. As there are, unfortunately, no separate models/schemas for `POST`ing a task request and `GET`ting info associated with a task, I'm afraid you can only document that in the description for now (and probably should).

Reply:
Also, perhaps it is unnecessary, here, to be explicit about what the `callback_url` can be used for. Possibly there will be other uses for it in the future.